test/integration/default_installation.bats

```diff
@@ -9,15 +9,10 @@ setup_file() {
 }
 
 @test "no startup warnings in logs" {
-    skip "Skip for 1.12.2 which has a known warning"
     run kubectl logs -n qdrant-helm-integration qdrant-0
     [ $status -eq 0 ]
     [[ "${output}" =~ .*INFO.* ]]
-    if [[ ! "${output}" =~ .*WARN.* ]]; then
-        echo "Found warning output:"
-        echo "${output}"
-        return 1
-    fi
+    [[ ! "${output}" =~ .*WARN.* ]]
 }
```

Comment:
Here I reverted the test change made in <https://github.com/qdrant/qdrant-helm/pull/262> because it's fixed in Qdrant 1.12.3.

Not entirely sure if reverting it is desired.

Reply:
@timvisee I think we can remove the `skip` as it is fixed now, but we can keep the additional `echo`'s to see directly what is wrong.

Reply:
The logic seems wrong here, it is written as if output does not contain `WARN`, then return 1 and echo output. While it should be if it contains `WARN` I would say, so:

```
    if [[ "${output}" =~ .*WARN.* ]]; then
```

This should fix the test as it currently errors without output that does not contain `WARN` 😛

Reply:
Patched in https://github.com/qdrant/qdrant-helm/pull/263/commits/8d9c4ce68b958235c5b92a34f04306e9584d778a